tests/acceptance/organization/contact-settings-test.js

```diff
@@ -0,0 +1,199 @@
+    }
+  });
+  const newName = "Mike";
+
+  stubOrganization(organizationData);
+  stubUser(securityOfficerData);
+  stubUser(billingContactData);
+  stubRequest('get', `${organizationApiUrl}/users`, function() {
+    return this.success({
+      _embedded: {
+        users: [{
+          id: 'some-bozo',
+          name: 'named bozo'
+        }, securityOfficerData, billingContactData]
+      }
+    });
+  });
+
+  stubRequest('put', organizationApiUrl, function(request) {
+    const body = this.json(request);
+    assert.equal(body.name, newName, 'name is correct');
+    assert.equal(body.security_alert_email, organizationData.security_alert_email, 'security alert email is correct');
+    assert.equal(body.ops_alert_email, organizationData.ops_alert_email, 'ops alert email is correct');
+    assert.equal(body.primary_phone, organizationData.primary_phone, 'primary phone is correct');
+    assert.equal(body.emergency_phone, organizationData.emergency_phone, 'emergency phone is correct');
+    assert.equal(body.address, organizationData.address, 'address is correct');
+    assert.equal(body.city, organizationData.city, 'city is correct');
+    assert.equal(body.state, organizationData.state, 'state is correct');
+    assert.equal(body.zip, organizationData.zip, 'zip is correct');
+    assert.equal(body.security_officer_id, securityOfficerData.id, 'security officer is correct');
+    assert.equal(body.billing_contact_id, billingContactData.id, 'billing contact is correct');
+    body.id = organizationId;
+    return this.success(body);
+  });
+
+  signInAndVisit(url);
+  fillInput('name', newName);
+  clickButton('Save');
+});
+
+test(`visiting ${url} and saving with error`, function(assert) {
+  assert.expect(1);
+
+  let securityOfficerData = buildUserData();
+  let billingContactData = buildUserData();
+  let organizationData = buildOrganizationData({
+    _links: {
+      self: {href: organizationApiUrl},
+      security_officer: {href: `/users/${securityOfficerData.id}`},
+      billing_contact: {href: `/users/${billingContactData.id}`},
+      users: {href: `${organizationApiUrl}/users`}
+    }
+  });
+  const newName = "Mike";
+
+  stubOrganization(organizationData);
+  stubUser(securityOfficerData);
+  stubUser(billingContactData);
+  stubRequest('get', `${organizationApiUrl}/users`, function() {
+    return this.success({
+      _embedded: {
+        users: [{
+          id: 'some-bozo',
+          name: 'named bozo'
+        }, securityOfficerData, billingContactData]
+      }
+    });
+  });
+
+  stubRequest('put', organizationApiUrl, function(request) {
+    return this.error(422, {
+      code: 422,
+      error: 'unprocessable_entity',
+      message: 'Name is not valid'
+    });
+  });
+
+  signInAndVisit(url);
+  fillInput('name', newName);
+  clickButton('Save');
+  andThen(function(){
+    const error = findWithAssert(':contains(There was an error)');
+    assert.ok(error.length > 0, 'Errors are on the page');
```

Comment:
elsewhere in tests for this codebase we use `find` and assert for truthiness of length:
```
const error = find('...');
assert.ok(error.length, 'msg');
```

It would be nice if `findWithAssert` actually did assert, but since it doesn't I don't usually use it when I'm finding an element to subsequently assert its presence.